src/components/modes/teacher/Settings.js

```diff
@@ -301,6 +344,12 @@ class Settings extends Component {
           </InputLabel>
           {selectControl}
         </FormControl>
+        <FormControl>
```

Comment:
Can you add a tooltip somewhere explaining what this does? E.g.:

```
Select the "public" option to make actions compatible with learning analytics applications.
```

Reply:
Once you hover over "Actions visbility"
![image](https://user-images.githubusercontent.com/45897168/88455864-ce920300-ce81-11ea-9af7-cc0b4ce1b61c.png)